### PR TITLE
Change user dialog

### PR DESCRIPTION
**Pull request by bramkragten**
image

### CODE REVIEW

**Comment by zsarnett** [comment]
Can you make the id `var(--secondary-text-color)`

File: src/panels/config/users/dialog-user-detail.ts [hunk 1]
```diff
@@ -55,31 +58,55 @@ class DialogUserDetail extends LitElement {
                 <div class="error">${this._error}</div>
               `
             : ""}
+          <div class="secondary">
+            ${this.hass.localize("ui.panel.config.users.editor.id")}: ${user.id}
+          </div>
+          <div>
```
Comment (zsarnett) [inline src/panels/config/users/dialog-user-detail.ts:64]: Why not use the `ha-chip` component?

File: src/panels/config/users/dialog-user-detail.ts [hunk 2]
```diff
@@ -55,31 +58,55 @@ class DialogUserDetail extends LitElement {
...
+              ? html`
+                  <span class="state"
+                    >${this.hass.localize(
+                      "ui.panel.config.users.editor.active"
+                    )}</span
+                  >
+                `
+              : ""}
+          </div>
           <div class="form">
```
Comment (zsarnett) [inline src/panels/config/users/dialog-user-detail.ts:93]: I think we should add `padding-top: 16px` to this element. Separate it from the not editable fields above

File: src/panels/config/users/dialog-user-detail.ts (hunk 1 above)
Comment (bramkragten) [reply src/panels/config/users/dialog-user-detail.ts:64]: Because chips are input elements

Comment (zsarnett) [reply src/panels/config/users/dialog-user-detail.ts:64]: Gotcha

**Pull request merged**